### ISSUE TITLE
Resources are not copied on MacOS.

### ISSUE THREAD

**Issue by renweizhukov** [mherrmann/fbs#39]
Followed [the manual of fman build system](https://build-system.fman.io/manual/) to place some resource files under `src/main/resources/`. Then ran the `python -m fbs freeze` command to copy the applicable files into my app's frozen directory inside the target/ folder. But found that all the resource files were not copied to the frozen directory.

Attached is the log of the debug output of `python -m fbs freeze`. 
[freeze.log](https://github.com/mherrmann/fbs/files/2544705/freeze.log)

This issue is not observed on Ubuntu 16.04LTS and Windows 10. I notice that there exists a directory `txt2mobi3` containing pyc files in the frozen directories on MacOS, while there doesn't exist such a directory on Ubuntu and Windows. Note that the resource files are supposed to be copied to a subdirectory `txt2mobi3/resources`.

You can find the source code at https://github.com/renweizhukov/txt2mobi3_app.

To repro this issue:

```
$ git clone https://github.com/renweizhukov/txt2mobi3_app.git
$ cd txt2mobi3_app
$ pip install -r requirements.txt
$ pip install PyInstaller
$ python -m fbs freeze
```

MacOS version:

```
ProductName:	Mac OS X
ProductVersion:	10.12.6
BuildVersion:	16G1618
```

Please let me know if you need any other info. Thank you very much for the help!

**Comment by mherrmann** [comment]
Hi,

firstly, the code you posted did not work for me because it uses the wrong `PyInstaller` and `PyQt5` versions. In particular, your `requirements.txt` lists `PyQt5 >= 5.11.3` when it should be `PyQt5 == 5.9.2`. And `pip install Pyinstaller` currently installs version 3.4, when it should be `3.3.1`.

On Mac, the resources are copied into the `target/txt2mobi3_app.app/Contents/`folder. I do find the contents of your [`resources folder`](https://github.com/renweizhukov/txt2mobi3_app/tree/master/src/main/resources) there.

Finally, another version problem: Your `requirements.txt` specifies `fbs >= 0.2.9.` This currently installs version 0.3.6. Since release [0.3.0](https://github.com/mherrmann/fbs/releases/tag/v0.3.0), the directory layout is different. Your project structure - in particular `src/main/resources/mac-frozen` and `.../linux-global` is wrong for this layout.

In short, I strongly believe you need to fix the library versions in your `requirements.txt`. For your current code, it needs to be:

    fbs==0.2.9
    PyQt5==5.9.2
    PyInstaller==3.3.1

Once you have moved `linux-global/` to `src/installer/linux` and `mac-frozen/` to `src/freeze/mac`, you can update to `fbs==0.3.6`.

Hope this helps.

**Comment by renweizhukov** [comment]
Thank you very much for the quick response. With your help, I was able to resolve the issue. I didn't know that "On Mac, the resources are copied into target/txt2mobi3_app.app/Contents/Resources/." 

One last question: I notice that there is a folder "target/txt2mobi3_app.app/Contents/MacOS/" which contains all the binaries and libraries. May I ask why the resources are not copied there?

**Comment by mherrmann** [comment]
That's because macOS `.app` bundles are supposed to have their binaries in `Contents/MacOS` and their data files in `Contents/Resources`. For more information, please see [here](https://developer.apple.com/library/archive/documentation/CoreFoundation/Conceptual/CFBundles/BundleTypes/BundleTypes.html#//apple_ref/doc/uid/10000123i-CH101-SW20).
